operator/charts/paladin-operator/assets/config.json

```diff
@@ -2,19 +2,30 @@
   "paladin": {
     "image": "{{ .Values.paladin.image.repository }}:{{ .Values.paladin.image.tag }}",
     "imagePullPolicy": "{{ .Values.paladin.image.pullPolicy }}",
-    "labels": {
-      "app": "paladin"
-    }
+    "labels": {{ toJson .Values.paladin.labels }},
```

Comment:
To be clear I think these need to be treated as _extra_ labels. The Operator should always use/prescribe a set of well-known labels i.e. `app.kubernetes.io/name` and `app.kubernetes.io/component` that it uses for consistent labeling for Service, Deployment, PDB, and NetworkPolicy selectors.

Reply:
I will add that code in the operator

Reply:
Added to the operator: https://github.com/LF-Decentralized-Trust-labs/paladin/pull/430/commits/321eb116d991b1a25a0c7c5e9ea93d870f1c436d#diff-9d910189a9a7a42fd5edc8ea7da4b1b89fd7f019dfc98ee19e30b3d124e9da12R348